data/strings/strings.txt

```diff
@@ -223,7 +223,7 @@
 
   [downloading]
     comment = Settings/Downloader - info for country which started downloading
-    tags = android, ios
```

Comment:
Is it really necessary to remove a space?

Reply:
yes, twine fails to detect the `ios` tag in the `tags = android, ios` string

Reply:
So it was our scripts' fault that introduced this extra space?

Reply:
yeap, without the space it works as expected